### ISSUE TITLE
Aria-required property is missing for required field in Dashboard screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#877]
**Describe the bug**
Aria-required property is missing for required field in Dashboard screen.

**To Reproduce**
1. Go to Dashboard screen (http://localhost:9992/cm/app/?view=dash)
2. Now, go to "Add Dashboard Gadgets" and add "Blogs" Dashboard gadget.
3. In the "Blogs" Dashboard gadget, on the right side, there is a icon "Click to create new blog". Click on that icon. A pop up opens.
4. "Blog link text" is a required field. Inspect the "Blog link text" text box element, you will find that the Aria-required=true property is missing for the input text.

**Expected behavior**
Aria-required=true property must be available for all required fields.

**Additional context**
Navigate all the options available for similar issue and fix.
